daprdocs/content/en/dotnet-sdk-docs/dotnet-client/dotnet-daprclient-usage.md

```diff
@@ -0,0 +1,149 @@
+---
+type: docs
+title: "DaprClient Usage"
+linkTitle: "DaprClient Usage"
+weight: 100000
+description: Essential tips and advice for using DaprClient
+---
+
+## Lifetime Management
+
+A `DaprClient` holds access to networking resources in the form of TCP sockets used to communicate with the Dapr sidecar. `DaprClient` implements `IDisposable` to support eager cleanup of resources.
+
+For best performance, create a single long-lived instance of `DaprClient` and provide access that shared instance throughout your application. `DaprClient` instances are thread-safe and intended to be shared.
+
+Avoid creating a `DaprClient` per-operation and disposing it when the operation is complete. 
+
+## Configuring DaprClient
+
+A `DaprClient` can configured by invoking methods on `DaprClientBuilder` class before calling `.Build()` to create the client. The settings for each `DaprClient` object are separate and cannot be changed after calling `.Build()`.
+
+```C#
+var daprClient = new DaprClientBuilder()
+    .UseJsonSerializerSettings( ... ) // Configure JSON serializer
+    .Build();
+```
+
+The `DaprClientBuilder` contains settings for:
+
+- The HTTP endpoint of the Dapr sidecar
+- The gRPC endpoint of the Dapr sidecar
+- The `JsonSerializerOptions` object used to configure JSON serialization
+- The `GrpcChannelOptions` object used to configure gRPC
+- The API Token used to authenticate requests to the sidecar
+
+The SDK will read the following environment variables to configure the default values:
+
+- `DAPR_HTTP_PORT`: used to find the HTTP endpoint of the Dapr sidecar
+- `DAPR_GRPC_PORT`: used to find the gRPC endpoint of the Dapr sidecar
+- `DAPR_API_TOKEN`: used to set the API Token
+
+### Configuring gRPC Channel Options
```

Comment:
Configuring gRPC channel options

Reply:
Btw @AaronCrawfis if you use `Insert a suggestion` feature like @mukundansundar - I'll be your friend forever 👍